ocs_ci/ocs/longevity.py

```diff
@@ -514,6 +535,73 @@ def validate_obcs_in_kube_job_reached_running_state(
 
         return obc_bound_list
 
+    def cluster_sanity_check(
+        self,
+        cluster_health=True,
+        db_usage=True,
+        resource_utilization=True,
+        disk_utilization=True,
+    ):
+        """
+        Cluster sanity checks
+
+        Args:
```

Comment:
These are all the default args and the default value of these arg is set to True. If we need to have all the mentioned checks done we can directly call the function cluster_sanity_check() without passing these args. 
I have categorized the different checks into buckets and calling those based on the bool value so that it can be more generic and useful to everyone. This way people can selectively choose the checks they wanted to perform and can avoid the other unnecessary checks by tuning the bool values.
For eg,If a test does not have any impact on the cpu/mem utilization, they can set the resource_utilization to False so that that code path is not called/executed.